Script/PRONTO.py

```diff
@@ -1222,6 +1274,8 @@ def main(argv):
 				extraction_hospital = ln.split('\t')[11]
 				tumor_content_nr = ln.split('\t')[12]
 				batch_nr = ln.split('\t')[13]
+				pathology_comment = ln.split('\t')[14]
```

Comment:
this can be handled in one go:

```python
[var1, var2, _, var3] = ln.split('\t')
```
the underscore is fields you don't want to parse into a variable 😉

Reply:
where are we in terms of backwards compatibility? Do we not want it? @xiaoliz0

Reply:
sorry, I should have been specific. I was more thinking about the backwards compatibility.

Reply:
Sorry Martin, I did not see your issue. Did you get any errors with your suggested scripts here? I tested with some previous data in our side, it works without problem. The "comment" will be empty.

Reply:
Ah, ok, so you have `pathology_comment` and `sample_info_comment` in all your metadata files? For us it currently looks like this:

```csv
Sample_id	Sequencing_run_id	Create_report	Year_of_birth	Year_of_diagnosis	Clinical_diagnosis	Sex	Study_id	Sample_material_id	Year_of_sample_collection	Requisition_hospital	Extraction_hospital	Tumor_content_pathology	Batch
```

I tested it earlier with a file like this and it seems that the missing headers cause PRONTO to fail.

Reply:
> Ah, ok, so you have `pathology_comment` and `sample_info_comment` in all your metadata files? For us it currently looks like this:
> 
> ```
> Sample_id	Sequencing_run_id	Create_report	Year_of_birth	Year_of_diagnosis	Clinical_diagnosis	Sex	Study_id	Sample_material_id	Year_of_sample_collection	Requisition_hospital	Extraction_hospital	Tumor_content_pathology	Batch
> ```
> 
> I tested it earlier with a file like this and it seems that the missing headers cause PRONTO to fail.

Emm, I tried to deleted the new added two "comment" headers  in the meta file for testing, it still works in our side. Could you take a check which header caused the fail in your side?

Reply:
When I use the header as I indicated, I get:

```bash
Traceback (most recent call last):
  File "/pronto/Script/PRONTO.py", line 1576, in <module>
    main(sys.argv[1:])
    ~~~~^^^^^^^^^^^^^^
  File "/pronto/Script/PRONTO.py", line 1269, in main
    pathology_comment = ln.split('\t')[14]
                        ~~~~~~~~~~~~~~^^^^
IndexError: list index out of range
```

Reply:
Ah, I might have misunderstood. You only deleted the new column headers not the whole columns? That seems like it would work. But in the old data we do not have the columns, right? At least at HUS we don't.

Reply:
Could we ignore this for the old data? The var will be empty.
```suggestion
				try:
					[DNA_sampleID, runID_DNA, _, ipd_birth_year, ipd_diagnosis_year, ipd_clinical_diagnosis_meta, ipd_gender, ipd_consent, DNA_material_id, ipd_collection_year, requisition_hospital, extraction_hospital, tumor_content_nr, batch_nr, pathology_comment, sample_info_comment] = ln.split('\t')
				except:
					pass
```

Reply:
I tried this solution but it fails for me because none of the vars are defined if the new columns are not present. Maybe this:
```suggestion
				ln = ln + (14-ln.count('\t'))
				[DNA_sampleID, runID_DNA, _, ipd_birth_year, ipd_diagnosis_year, ipd_clinical_diagnosis_meta, ipd_gender, ipd_consent, DNA_material_id, ipd_collection_year, requisition_hospital, extraction_hospital, tumor_content_nr, batch_nr, pathology_comment, sample_info_comment] = ln.split('\t')
```
This would simply add tabs to the string to fill it up to 15 columns prior to splitting it. Would something like this be ok?

Reply:
Just tried it and it works both with the old data and the new.

Reply:
I created a PR which also contains the changes you made in your other PR: https://github.com/InPreD/PRONTO/pull/63

Reply:
done 👍

Reply:
Great! Could I merge this PR now? Then we could move further for a new version.